doc/examples/segmentation/plot_regionprops.py

```diff
@@ -97,32 +97,46 @@
 mask = morphology.remove_small_holes(mask, 50)
 labels = measure.label(mask)
 
-fig = px.imshow(img, binary_string=True)
-fig.update_traces(hoverinfo='skip')  # hover is only for label info
+property_names = ["label", "area", "eccentricity", "perimeter", "intensity_mean"]
 
+# Get region properties
 props = measure.regionprops(labels, img)
-properties = ['area', 'eccentricity', 'perimeter', 'intensity_mean']
 
-# For each label, add a filled scatter trace for its contour,
-# and display the properties of the label in the hover of this trace.
-for index in range(1, labels.max()):
-    label_i = props[index].label
+# Filter out big regions
+filtered_props = [prop for prop in props if prop.area < 5000]
```

Comment:
This doesn't feel like a good way to address this. Maybe we should replace the threshold based approach with an one based on sobel and watershed? E.g. like in https://github.com/scikit-image/scikit-image/blob/779ae8a910e04bebf7be76672bcaed3ad3b6f3d3/doc/examples/segmentation/plot_expand_labels.py#L24-L35

What do you think?

Reply:
I can include this. imo the segmentation method doesn't matter for this example as the motivation seems to be to use plotly for an interactive hover info. 

If I include the sobel + watershed here, would it make the other example redundant?